starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,34 @@
 // Iteration 1
 // 1. Create an array of strings
 
+var tasksList: Array<string> = [];
+
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(nameTask:string){
```

Comment:
this function should expect also a type of number:
`function addTask (task: string): number {
`